packages/forma-36-react-components/src/components/Dropdown/DropdownContainer/DropdownContainer.tsx

```diff
@@ -1,252 +1,131 @@
-import React, { Component } from 'react';
+import React, { forwardRef, useCallback, useEffect, useRef } from 'react';
 import cn from 'classnames';
 import ReactDOM from 'react-dom';
-import InViewport from '../../InViewport';
-import { positionType, AnchorDimensionsAndPositonType } from '../Dropdown';
 
+import { positionType } from '../Dropdown';
 import styles from './DropdownContainer.css';
 
-export interface DropdownContainerProps {
-  onClose?: Function;
-  dropdownAnchor?: HTMLElement | null;
-  className?: string;
+export interface DropdownContainerProps
+  extends React.HTMLAttributes<HTMLElement> {
   children?: React.ReactNode;
-  testId?: string;
-  openSubmenu?: (value: boolean) => void;
-  anchorDimensionsAndPositon?: AnchorDimensionsAndPositonType;
-  position: positionType;
+  className?: string;
   getRef?: (ref: HTMLElement | null) => void;
+  isOpen: boolean;
+  onClose?: Function;
+  openSubmenu?: (value: boolean) => void;
+  position?: positionType;
   submenu?: boolean;
-  width?: number | false;
-  isAutoalignmentEnabled?: boolean;
-}
-
-export interface DropdownState {
-  dropdownDimensions: {
-    width: number;
-    height: number;
-  };
-  position: positionType;
+  testId?: string;
 }
 
-const defaultProps: Partial<DropdownContainerProps> = {
-  testId: 'cf-ui-dropdown-portal',
-  position: 'bottom-left',
-  submenu: false,
-  isAutoalignmentEnabled: true,
```

Comment:
@denkristoffer Since you made the prop work again, I think you'd want to keep the default value of `true` for this prop and also add it back to the `DropdownContainerProps` interface above, no?

Reply:
The functionality has been moved to `Dropdown.tsx`, so you shouldn't ever want/have to pass it to `DropdownContainer`.